Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -47,47 +51,53 @@ $ curl -X GET http://10.109.65.142:9080/metrics 2>/dev/null | grep csi
 csi_liveness 1
 ```
 
-Check the [monitoring doc](../Monitoring/ceph-monitoring.md) to see how to integrate CSI
-liveness and grpc metrics into ceph monitoring.
+Check the [monitoring documentation](../Monitoring/ceph-monitoring.md) to see how to integrate CSI
+liveness and GRPC metrics into Ceph monitoring.
 
 ## Dynamically Expand Volume
 
-### Prerequisite
+### Prerequisites
 
 * For filesystem resize to be supported for your Kubernetes cluster, the
-  kubernetes version running in your cluster should be >= v1.15 and for block
-  volume resize support the Kubernetes version should be >= v1.16. Also,
-  `ExpandCSIVolumes` feature gate has to be enabled for the volume resize
+  Kubernetes version must be >= v1.15 and for block
```

Comment:
This paragraph is obsolete since our minimum supported version is now past K8s 1.20. Let's remove this paragraph.